x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -327,7 +327,7 @@ protected static List<TestCaseSupplier> anyNullIsNull(boolean entirelyNullPreser
                         oc.evaluatorToString,
                         oc.expectedType,
                         nullValue(),
-                        oc.getExpectedWarnings(),
+                        null,
```

Comment:
Because the warnings were just copied when the `null` parameter testcase was generated, instead of just checking the `null` value is returned, the warnings were also checked and the tests failed where in fact there were no warnings expected.